src/provider/camunda-platform/properties/ImplementationProps.js

```diff
@@ -23,11 +22,11 @@ import {
 
 
 export function ImplementationProps(props) {
-  const {
+  let {
```

Comment:
I can't see this overwritten anywhere. I believe we can leave it as `const`

Reply:
I haven't noticed it was already merged 🙈

Reply:
NVM: https://github.com/bpmn-io/bpmn-js-properties-panel/pull/519